### ISSUE TITLE
Clearer shortcuts with an expansible gui

### ISSUE THREAD

**Issue by Liger0** [mgba-emu/mgba#1729]
Hi there, I see that the settings tab in general isn't very clean, as there is no way to expand the window, and everything isoverlapped by default, as seen here
![screen](https://user-images.githubusercontent.com/20385116/79276469-5b531b00-7ea8-11ea-8b5b-cb3eed97814b.png)

I think that making the menus not overlap by default and making it possible to expand the window would be cool.
It also is hard to follow the shortcuts.
For example,  I'll take the "turbo mode". Near to its name there is the letter "F" but it doesn't say what that means.
As a button there is Shift+Tab, but that is only for the permanent feature and not for the temporary one which is Tab alone. I think these options should be separated and it should be able to see how to set the temporary turbo mode.
The same probably applies for other features.

**Comment by endrift** [comment]
Some of these issues are specific to the Italian translation. However, there are issues with the window size I want to address.

**Comment by endrift** [comment]
I believe this is somewhat improved in latest dev (ignoring translation issues, since I still haven't had anyone update the italian translation), but I'm not sure _how much_ better it is.
